src/defaults/mod.rs

```diff
@@ -63,9 +63,11 @@ defaults! {
 /// passwd_timeout and timestamp_timeout.
 fn fractional_minutes(input: &str) -> Option<i64> {
     if input.contains('.') {
+        //math note: the token length is capped at 18 characters, so the max value parsed is 1e17;
+        //1e17 * 60 still fits in an i64, so the 'as' can never cause truncation
```

Comment:
Isn't the max value parsed `1e18 - 1` if you write `99999999999.` (with 17 9s)? By the way is it possible for `fractional_minutes` to get something like `1e100` (or even `inf` or `NaN`) as input? `f64::from_str` accepts that just fine.

Reply:
Hahaha.

- `inf` or `NaN` will not be possible.
- the Numeric tokenizer will allow hex digits and dots, so indeed "1e100" would work. I don't even think that's a bad thing, but it does mean that this comment is incorrect.

Reply:
Replaced this line in 0b1386d